firebase-common/src/jsMain/kotlin/dev/gitlive/firebase/externals.kt

```diff
@@ -356,6 +356,7 @@ external object firebase {
             fun where(field: Any, opStr: String, value: Any?): Query
             fun onSnapshot(next: (snapshot: QuerySnapshot) -> Unit, error: (error: Error) -> Unit): () -> Unit
             fun limit(limit: Double): Query
+            fun orderBy(field: Any, direction: Any): Query
```

Comment:
can add the overloads you expect here - so multiple orderBy functions with String/FieldPath

Reply:
There seems to be two different FieldPath's `dev.gitlive.firebase.firebase.FieldPath` and `dev.gitlive.firebase.FieldPath`. With the first being available in `externals.kt` and the latter available in `jsMain/.../firestore.kt`